addons/auth_passkey/views/auth_signup_login_templates.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template priority="8" id="auth_passkey_login" inherit_id="web.login" name="Sign up - Reset Password">
+        <xpath expr="//div[hasclass('o_login_auth')]/parent::div" position="replace">
```

Comment:
I wouldn't do a replace here, but a position="after", to avoid have to copy/paste the code from `<template id="web.login" name="Login">`

e.g. that will avoid the copy/paste on the below `Log in as superuser`.